client/src/components/Ratings/Ratings.jsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
 import styled from 'styled-components';
-import { RatingProvider } from '../../contexts/ReviewMeta';
+import { useMeta } from '../../contexts/ReviewMeta';
 
 import RatingBreakdown from './RatingBreakdown';
 import ReviewList from './ReviewList';
 import { ReviewProvider } from '../../contexts/ProductReview';
 
 export default function Ratings() {
+  const currentMeta = useMeta();
+  const ratingBreakdown = (currentMeta && currentMeta.avgRating) ? <RatingBreakdown /> : <div />;
```

Comment:
Interesting, is this a placeholder div in case it's loading/not available? I might steal this.

Reply:
Yep. It's a lazy way of double conditional render - This handles the edge case of when a product has no review and also bypass the async loading error of getting undefined